src/App.jsx

```diff
@@ -14,57 +16,77 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [val, setGood] = useState(goods[8]);
+  const [title, setTitle] = useState(`${val} is selected`);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {`${title}`}
```

Comment:
That's what i meant.
```suggestion
        {`${val} is selected`}
```